app/routes/projects/index.tsx

```diff
@@ -155,20 +198,20 @@ export default function Projects() {
         <div className="homeWrapper__navbar">
           <div className="homeWrapper__navbar__tabs">
             <div
-              className={`homeWrapper__navbar__tabs--title ${getTabClass("ideas")}`}
-              onClick={() => handleTabChange("ideas")}
+              className={`homeWrapper__navbar__tabs--title ${getTabClass(ideasTab.name)}`}
+              onClick={() => handleTabChange(ideasTab.name)}
             >
               Ideas
             </div>
             <div
-              className={`homeWrapper__navbar__tabs--title ${getTabClass("activeProjects")}`}
-              onClick={() => handleTabChange("activeProjects")}
+              className={`homeWrapper__navbar__tabs--title ${getTabClass(activeProjectsTab.name)}`}
+              onClick={() => handleTabChange(activeProjectsTab.name)}
             >
               Active Projects
             </div>
             <div
-              className={`homeWrapper__navbar__tabs--title ${getTabClass("myProposals")}`}
-              onClick={() => handleTabChange("myProposals")}
+              className={`homeWrapper__navbar__tabs--title ${getTabClass(myProposalsTab.name)}`}
+              onClick={() => handleTabChange(myProposalsTab.name)}
             >
```

Comment:
In this part I see repeated code, can it be possible to make a tabs object so as not to repeat the code?

Reply:
Yes, I agree, maybe we can do something like this:
```
const tabs = [
  activeProjectsTab,
  myProposalsTab,
  ideasTab,
]

{
  tabs.map(tab => (
    <div
      className={`homeWrapper__navbar__tabs--title ${getTabClass(tab.name)}`}
      onClick={() => handleTabChange(tab.name)}
      key={tab.name}
    >
      {tab.title}
    </div>
  ))
}
```

Reply:
Thanks i did the change